cli/build.rs

```diff
@@ -13,7 +13,7 @@ fn main() -> Result<()> {
     // HACK: used by Nix, since at the moment
     // the checks are a process that's hard to accomodate
     // in Nix environment
-    if std::option_env!("IROHA_SKIP_WASM_CHECKS").is_none() {
+    if std::option_env!("IROHA_SKIP_WASM_CHECKS").is_some() {
         check_default_executor()?;
     }
```

Comment:
```suggestion
```

IMO we should remove this completely

Reply:
we build it manually, you can remove it

Reply:
Ok then I will remove the env variable checking from both files

Reply:
no, just this one

Reply:
or if it's possible make sure that these smartcontracts are built only when running `cargo test`

Reply:
Ok I will try test cargo feature

Reply:
what about checking if the PROFILE variable is set to test? would that be okay?

Reply:
There's no `test` profile, `cfg(test)` is actually orthogonal to the profile used (meaning, you can run unit tests in release profile).

Also, I an not sure that `cfg(test)` will work the intended way in `build.rs`..